logging.go

```diff
@@ -66,3 +66,14 @@ func SetFacilityLogLevel(f, l string) {
 	*/
 	logging.SetLogLevel(f, l)
 }
+
+// IsLogLevel checks if given string is a valid log level.
+func IsLogLevel(s string) bool {
+	for _, level := range []string{"DEBUG", "CRITICAL", "ERROR", "WARNING", "NOTICE", "INFO"} {
+		if s == level {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
What a valid log level is is defined by the underlying library. We should not define that. It already provides methods to check whether a level exists: https://godoc.org/github.com/ipfs/go-log#LevelFromString

Reply:
This requires to update go-log. Studying go-log and zap to make sure formatting stays the same.

Reply:
> Studying go-log and zap to make sure formatting stays the same.

I could not find a way to keep the formatting same. It seems that new version(version in which LevelFromString exist) provide only two formatting option colorful or plain. What we use right now, logging.LogFormats, is removed.
New version uses zap internally. I think using zap.Config and zapcore.EncodeConfig it would be possible to achieve the right formatting, but it is not possible to provide that using go-log.

Reply:
As discussed, we do not need to verify that the log level is valid since the library does that and errors accordingly: https://github.com/ipfs/go-log/blob/v0.0.1/oldlog.go#L129